backend/keystore/keystore.go

```diff
@@ -48,14 +48,10 @@ type Keystore interface {
 	// https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#key-identifiers
 	RootFingerprint() ([]byte, error)
 
-	// CosignerIndex returns the index at which the keystore signs in a multisig configuration.
-	// The returned value is always zero for a singlesig configuration.
-	CosignerIndex() int
-
 	// SupportsAccount returns true if they keystore supports the given coin/account.
 	// meta is a coin-specific metadata related to the account type.
 	// If false, the backend will add one account per supported script type.
```

Comment:
> If false, the backend will add one account per supported script type.

Not sure I understand the comment. What is `false`, the returned value? But then "backend will add ..." doesn't concern this `SupportsAccount` method at all. Or did it refer to the `multisig` bool?

Reply:
No, it means that if this function returns false, that the backend will split the accounts, one per script type. I know it does not concern this function, but thought it might be helpful. Can also delete it :shrug: